### ISSUE TITLE
Could not find a declaration file for module 'react-wavify'

### ISSUE THREAD

**Issue by haziqfiqri** [woofers/react-wavify#70]
After installation with my React project using ``` npm install react-wavify --save``` . I tried testing react wavify inside one of my components. However it gives me an error below when I tried to test it just like in the npm react wavify page. 

```
Could not find a declaration file for module 'react-wavify'. 'E:/work/node_modules/react-wavify/dist/react-wavify.min.js' implicitly has an 'any' type.
```

package.json
```
  "dependencies": {
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-scripts": "3.4.1",
    "react-typed": "^1.2.0",
    "react-wavify": "^1.2.3"
  },
  "devDependencies": {
    "@material-ui/core": "^4.9.8",
    "@material-ui/icons": "^4.9.1",
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.5.0",
    "@testing-library/user-event": "^7.2.1",
    "@types/jest": "^24.9.1",
    "@types/node": "^12.12.30",
    "@types/react": "^16.9.25",
    "@types/react-dom": "^16.9.5",
    "typescript": "^3.7.5"
  }
```

**Comment by haziqfiqri** [comment]
Vscode giving me hint of the problem its from the import itself, but Im not sure what I could do with it.
```
> 5 | import Wave from 'react-wavify'
```
I tried solution like ```npm install @types/react-wavify``` and recheck whether the react-wavify is in the node_module, yes it does but still not solving error I'm currently facing. Below are sample code of how I test to use the react-wavify component inside my component.

```
import React from 'react'
import { Container, makeStyles } from '@material-ui/core'
import Wave from 'react-wavify'

const useStyles = makeStyles(theme => ({
    heroContent: {
        backgroundColor: theme.palette.background.paper,
        padding: theme.spacing(8, 0, 6),
    },
}))

const Hero: React.FC = () => {
    const classes = useStyles()
    return (
        <React.Fragment>
            <div className={classes.heroContent}>
                <Container maxWidth="sm">
                    <Wave fill='#f79902'
                        paused={false}
                        options={{
                        height: 20,
                        amplitude: 20,
                        speed: 0.15,
                        points: 3
                        }}
                    />
                </Container>
            </div>
        </React.Fragment>
    )
}

export default Hero
```

**Comment by woofers** [comment]
Thanks for reporting this issue @haziqfiqri.   I was able to reproduce this using a simple TypeScript project.  

It appears that I need to configure some TypeScript definitions for this package which I plan on adding in a new release.

However I was able to workaround this issue by adding `"noImplicitAny": false` to the `"compilerOptions"` to my `tsconfig.json`:
```json
{
  "compilerOptions": {
    "noImplicitAny": false
  }
}
```
This allowed me use the package with TypeScript and I would recommend trying in the meantime.
